src/components/AboutSection.astro

```diff
@@ -11,10 +11,8 @@ import Section from "./Section.astro";
       <h1>Kharagpur Open Source Society</h1>
     </div>
     <p class="about-description">
-      We are a group of Open Source Enthusiasts who focus on something more preliminary and relevant, 'A Love for
-      Coding'. Every year KOSS plans to hold events for familiarizing students with UNIX tools, Linux environment, Git
-      Development Workflow, practice Algorithms and Data Structures, besides conducting workshops on GUI programming,
-      Web development, Android application Development and Hackathons.
+	A community of open-source enthusiasts at IIT Kharagpur, promoting open-source and FOSS through workshops, seminars, and events. Fostering a culture of technology and open source, we encourage learning, mentorship, and diverse software engineering projects.
+	With a focus on a love for coding, KOSS members pursue innovative ideas, from solving real-world problems to fun projects, while supporting each other's growth and expertise. Every year, we hold events to familiarise students with UNIX tools, the Linux environment, Git Development Workflow, algorithms, and data structures.
```

Comment:
You will have to add a `<br />` element to introduce the line break. A normal newline does not work.
![image](https://github.com/kossiitkgp/YAKW/assets/34770591/1cb45aab-c6a6-4e46-85ce-e65f61e393b4)

Reply:
Please review the changes